README.rst

```diff
@@ -14,381 +14,62 @@
 
-
-Ray Serve Quick Start
----------------------
-
-.. image:: https://raw.githubusercontent.com/ray-project/ray/master/doc/source/serve/logo.svg
-  :width: 400
-
-`Ray Serve`_ is a scalable model-serving library built on Ray. It is:
-
-- Framework Agnostic: Use the same toolkit to serve everything from deep
-  learning models built with frameworks like PyTorch or Tensorflow & Keras
-  to Scikit-Learn models or arbitrary business logic.
-- Python First: Configure your model serving declaratively in pure Python,
-  without needing YAMLs or JSON configs.
-- Performance Oriented: Turn on batching, pipelining, and GPU acceleration to
-  increase the throughput of your model.
-- Composition Native: Allow you to create "model pipelines" by composing multiple
-  models together to drive a single prediction.
-- Horizontally Scalable: Serve can linearly scale as you add more machines. Enable
-  your ML-powered service to handle growing traffic.
-
-To run this example, you will need to install the following:
-
-.. code-block:: bash
-
-    $ pip install scikit-learn
-    $ pip install "ray[serve]"
-
-This example runs serves a scikit-learn gradient boosting classifier.
-
-.. code-block:: python
-
-    import pickle
-    import requests
-
-    from sklearn.datasets import load_iris
-    from sklearn.ensemble import GradientBoostingClassifier
-
-    from ray import serve
-
-    serve.start()
-
-    # Train model.
-    iris_dataset = load_iris()
-    model = GradientBoostingClassifier()
-    model.fit(iris_dataset["data"], iris_dataset["target"])
-
-    @serve.deployment(route_prefix="/iris")
-    class BoostingModel:
-        def __init__(self, model):
-            self.model = model
-            self.label_list = iris_dataset["target_names"].tolist()
-
-        async def __call__(self, request):
-            payload = (await request.json())["vector"]
-            print(f"Received flask request with data {payload}")
-
-            prediction = self.model.predict([payload])[0]
-            human_name = self.label_list[prediction]
-            return {"result": human_name}
-
-
-    # Deploy model.
-    BoostingModel.deploy(model)
-
-    # Query it!
-    sample_request_input = {"vector": [1.2, 1.0, 1.1, 0.9]}
-    response = requests.get("http://localhost:8000/iris", json=sample_request_input)
-    print(response.text)
-    # Result:
-    # {
-    #  "result": "versicolor"
-    # }
-
-
-.. _`Ray Serve`: https://docs.ray.io/en/master/serve/index.html
-
 More Information
 ----------------
 
 - `Documentation`_
-- `Tutorial`_
-- `Blog`_
-- `Ray 1.0 Architecture whitepaper`_ **(new)**
-- `Exoshuffle: large-scale data shuffle in Ray`_ **(new)**
+- `Ray Architecture whitepaper`_
+- `Exoshuffle: large-scale data shuffle in Ray`_
 - `RLlib paper`_
-- `RLlib flow paper`_
 - `Tune paper`_
 
 *Older documents:*
 
 - `Ray paper`_
 - `Ray HotOS paper`_
 
+.. _`Ray AIR`: https://docs.ray.io/en/master/ray-air/getting-started.html
```

Comment:
These currently all link to `master`. We might want to change to `latest` after 2.0 gets released to avoid confusion.

Reply:
I changed them now (they all work).